platform/resources/runJobOnWorker.groovy

```diff
@@ -0,0 +1,7 @@
+import hudson.model.*
+
+JOB_TO_RUN = build.environment.get('JOB_TO_RUN')
+ghprbActualCommit = build.environment.get('ghprbActualCommit')
+WORKER_LABEL = build.environment.get('WORKER_LABEL')
+
+build(JOB_TO_RUN, ghprbActualCommit: ghprbActualCommit, WORKER_LABEL: WORKER_LABEL)
```

Comment:
@jzoldak  This isn't part of the job dsl, it is part of the build flow dsl (see https://wiki.jenkins-ci.org/display/JENKINS/Build+Flow+Plugin). We use it in our testeng-ci scripts (ex: https://github.com/edx/testeng-ci/blob/master/jenkins/flow/pr/edx-platform-bok-choy-pr.groovy)